container.go

```diff
@@ -279,6 +279,10 @@ func (c *Container) Snapshots() ([]Snapshot, error) {
 
 // Caller needs to hold the lock
 func (c *Container) state() State {
+	if c.container == nil {
```

Comment:
Ghost catch. Must've been triggered by a change in lxd at some point, I assume. Also, one more reason languages shouldn't have `nil` as a concept.